app/views/online-class/SchedulerView.vue

```diff
@@ -80,6 +90,14 @@ export default {
       return [lang, codeLang, level, dateTime].join(' &bull; ')
     }
   },
+  created () {
+    getGoogleCalendarSync().then(res => {
```

Comment:
how long does it take for us to sync and API to respond? Will it ever become greater than a minute?

Reply:
about.. 1s or 2s from china.  i don't think it would greater than 1 min, in global server I believe it would less than 1s